docs/docs/timelock-encryption.md

```diff
@@ -0,0 +1,119 @@
+Additionally, this means that users can decrypt their timelock ciphertext *offline*, rather than requiring submission to a third party to do it for them.
+
+## I don't have time to read all of this
+
+- [here is our blog post announcing the new network and timelock encryption](https://drand.love/blog/2023/03/28/timelock-on-fastnet/)
+- [here is Yolan's talk at Real World Crypto](https://www.youtube.com/watch?v=Xh849Ij3lhU) on timelock encryption
+- [here is the paper on eprint](https://eprint.iacr.org/2023/189)
+
+
+## ⚠️ Security assumptions ⚠️
+
+While our timelock scheme has been formally audited by reknowned security firm [Kudelski Security](https://kudelskisecurity.com)[1], as with any cryptographic scheme, it's important to understand its limitations before you go and bet the farm on it! This is not an exhaustive list, and you ought to also read the [drand security model](https://drand.love/docs/security-model/), but it's a good place to start.
+
+- **there have never been a threshold of malicious nodes in the drand network**
+
+If there were, at any time, a threshold number of malicious nodes in the network, they could cooperatively generate all future random values and use them to decrypt arbitrary future timelock ciphertexts. Drand randomness, while unpredictable, is entirely deterministic, even after the resharing of keys.
+That said, `fastnet` (our new network for timelock encryption) was started with 16 different participants running 23 nodes, so we're confident a threshold number of them have not been compromised!
+
+- **neither the scheme, nor drand are quantum resistant**
+
+The cryptography we employ do not use quantum resistant algorithms. If you timelock encrypt something for 1000 years and a viable quantum computer becomes available, anybody can decrypt it. Right now there are no widespread quantum resistant schemes for threshold cryptography, but we're keeping an eye out for the future!
+To that end, consider it relatively safe to timelock encrypt something for 5-20 years depending on how bullish you are on quantum computing.
+
+- **if the League of Entropy shuts down, members will delete their keys**
+
+The world is unpredictable (just like drand randomness... heh), and it's possible that the League of Entropy will all hang up their coats at some point in the future. Should that happen, members would have two choices with their private keys: release them to the world or delete them entirely.
+The former would mean that ciphertexts created for some time after the cessation of the network would be decryptable to everybody. The latter would mean that ciphertexts created for some time after the cessation of the network would be unencryptable forever (/until quantum computers can break them).
+In the interests of privacy, we felt the latter option was preferable. That said, if you encrypt the private key to your Filecoin fortune to stop yourself from spending it now and the network stops... you're going to have a bad time.
```

Comment:
```suggestion
In the interests of privacy, we felt the latter option was preferable. That said, if you encrypt the private key to your [insert cryptocurrency name] fortune to stop yourself from spending it now and the network stops... you're going to have a bad time.
```

Reply:
ffs I can't believe you're removing this top shill opportunity :p